docs/_plugins/3rd-party-integration/leaflet-editinosm.md

```diff
@@ -4,7 +4,7 @@ category: 3rd-party-integration
 repo: https://github.com/yohanboniface/Leaflet.EditInOSM
 author: Yohan Boniface
 author-url: https://yohanboniface.me/
-demo: 
+demo: http://yohanboniface.github.io/Leaflet.EditInOSM/examples/basic.html
```

Comment:
Quite a few demos like this one are blocked due to [Mixed Content](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content). I suppose there's a higher chance of the authors fixing these problems if a demo is linked to but does not work, rather than not being linked to at all?

Reply:
I think it's just recently that Chrome auto-upgrades http:// to https://, not sure why they're not doing that for subresources.

Reply:
In the meanwhile, me, in a time-machine:

https://github.com/yohanboniface/Leaflet.EditInOSM/issues/15